common/scala/src/main/scala/whisk/common/TransactionId.scala

```diff
@@ -55,7 +55,14 @@ case class TransactionId private (meta: TransactionMetadata) extends AnyVal {
    */
   def mark(from: AnyRef, marker: LogMarkerToken, message: String = "", logLevel: LogLevel = InfoLevel)(
     implicit logging: Logging) = {
-    logging.emit(logLevel, this, from, createMessageWithMarker(message, LogMarker(marker, deltaToStart)))
+
+    if (meta.metricsLog) {
+      logging.emit(logLevel, this, from, createMessageWithMarker(message, LogMarker(marker, deltaToStart)))
+    }
```

Comment:
This is a dangerous change in behavior. I'd rather have this flagged by additional information like so:

```scala
if (message.nonEmpty) {
  val print = if(meta.metricsLog) createMessageWithMarker(message, LogMarker(marker, deltaToStart)) else message
  logging.emit(logLevel, this, from, print)
}

Assuming that you still want the message to be printed if it was given by the developer (we should probably clean that up later and log the lines explicitly vs. implicitly through this class, but it keeps the changes small for now)

Reply:
I changed the PR (via amend) to emit simple log message in the original debug level to avoid loss of information. I will tackle log reduction in a following commit